website/content/en/highlights/2021-08-24-helm-vector-dev.md

```diff
@@ -0,0 +1,30 @@
+---
+date: "2021-08-24"
+title: "Introducing https://helm.vector.dev"
+description: "A new home for Vector's Helm charts"
+authors: ["spencergilbert"]
+pr_numbers: []
+release: "0.16.0"
+hide_on_release_notes: false
+badges:
+  type: "deprecation"
+  platforms: ["helm"]
+---
+
+Vector’s 0.16.0 release will be the last version that publishes charts to both
+https://packages.timber.io/helm/latest and https://packages.timber.io/helm/nightly repositories.
+
+The new repository contains all released charts from the previous `latest` repository.
+Moving forward we will be releasing charts at their own pace as we work towards the stable
+releases for the _vector-agent_ and _vector-aggregator_ charts.
+
+Development and issue tracking will be migrated to https://github.com/timberio/helm-charts
+in the coming days.
+
+## Upgrade Guide
```

Comment:
Should we also mention how the install commands will differ? I believe it will now be `helm install vector/vector-agent`? That redundancy is also making me wonder if it shouldn't just be `helm install vector/agent`. I'm curious what you think of that.

What will migration look like if they have the existing helm chart already installed?

Reply:
_maybe_ but breaking changes! Also, datadog agent is installed with `datadog/datadog`! - note that the name there is only used locally, so you could add it as `jesse-loves-helm` and install `jesse-loves-helm/vector-agent`

Reply:
Ah, gotcha, I didn't realize the name was whatever you want. I think keeping `vector-agent` makes sense.